### DISCUSSION TITLE
import export settings

### DISCUSSION THREAD

**Discussion by kinghat** [ImranR98/Obtainium#1261] [General]
i imported settings from another install but i no longer see the import/export settings on that tab :thinking: 

![image](https://github.com/ImranR98/Obtainium/assets/16073360/ccbd1983-58ee-44e5-823a-146974b07f26)

can create an issue from this if this is a bug.

**Comment by ImranR98** [comment]
You need to pick the export directory first

**Reply by kinghat** [reply]
ok that worked. i had already crated a directory and selected it and there was an auto export created as well. must have been some order of operations that messed things up.
